packs/feats/psi-development.json

```diff
@@ -36,6 +36,84 @@
                 "priority": 10,
                 "value": 1
             },
+            {
+                "key": "ActiveEffectLike",
+                "mode": "add",
+                "path": "flags.pf2e.psychic.dedication.psiCantrips",
+                "predicate": [
+                    "feature:the-distant-grasp"
+                ],
+                "priority": 10,
+                "value": {
+                    "label": "PF2E.SpecificRule.Psychic.PsiCantripLabels.TelekineticRend",
+                    "value": "telekinetic-rend"
+                }
+            },
```

Comment:
Each of these should go on their respective mind, predicated on having this feat.

Reply:
I realize this just reinvents what we had before. I think we should figure out the bug then fix that and revert to the old way.

Reply:
> I realize this just reinvents what we had before. I think we should figure out the bug then fix that and revert to the old way.

Yeah, the idea was that since the predicates were not being read in the Choice Set, they had to be processed before the cantrips even went into the array, so that's why I did it this way. But if the reason the original way is not working is a bug, then I will try to get some reproduceable steps and make an issue about it, and once it's fixed revert, as you say.

Reply:
Drafting this for now. I've reported the bug in #14574.